WordGame/Constant/Constant.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Constant.swift
+//  WordGame
+//
+//  Created by Kumari Ritu Pal on 25/07/22.
+//
+
+import Foundation
+import UIKit
+
```

Comment:
You can make each of these structs as separate enum and use them. Static properties inside Structs are good but structs inside structs are expensive. Ultimately you are holding static properties inside structs. So in my opinion, enums can be better for this purpose.
For example : 
``` swift 
enum Constants {
    static let maxWordPairForAGame = 15
    static let maxWrongLimit = 3
    static let maxTimeForAttempt = 5.0
    
    enum HomeScreen {
        static let correctAttempts = "home_correct_attemps".localized()
        static let wrongAttempts = "home_wrong_attemps".localized()
        static let correct = "home_correct_text".localized()
        static let wrong = "home_wrong_text".localized()
    }

    enum Styles {
        static let mainColor: UIColor = #colorLiteral(red: 0.9294117647, green: 0.431372549, blue: 0.262745098, alpha: 1)
        static let buttonTextColor: UIColor = .white
        static let backGroundColor: UIColor = .white
        static let tabbarStyleColor: UIBarStyle = .black

    }
    
    enum Alert {
        static let title = "alert_title".localized()
        static let message = "alert_message".localized()
        static let leftButtonTitle = "alert_no_title".localized()
        static let rightButtonTitle = "alert_yes_title".localized()
        static let points = "points".localized()
    }
}
```